node/src/test/scala/coop/rchain/node/perf/HistoryGenKeySpec.scala

```diff
@@ -0,0 +1,443 @@
+package coop.rchain.node.perf
+
+import cats.Parallel
+import cats.effect.{Concurrent, ContextShift, Sync}
+import cats.syntax.all._
+import coop.rchain.crypto.hash.Blake2b256
+import coop.rchain.metrics.{Metrics, NoopSpan, Span}
+import coop.rchain.rspace.hashing.Blake2b256Hash
+import coop.rchain.rspace.history.HistoryMergingInstances.{CachingHistoryStore, MergingHistory}
+import coop.rchain.rspace.history.RadixTree.ExportData
+import coop.rchain.rspace.history._
+import coop.rchain.rspace.history.instances._
+import coop.rchain.shared.syntax._
+import coop.rchain.shared.{Base16, Log, Stopwatch}
+import coop.rchain.store.{InMemoryKeyValueStore, KeyValueStore, LmdbStoreManager}
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import scodec.bits.ByteVector
+
+import java.io.File
+import java.math.BigInteger
+import java.nio.file
+import java.nio.file.Files
+import scala.reflect.io.{Directory, Path}
+import scala.util.Random
+
+class HistoryGenKeySpec extends FlatSpec with Matchers with BeforeAndAfterAll {
+
+  object TypesOfHistory extends Enumeration {
+    type TypeHistory = Value
+    val Merging, Radix, Default = Value
+  }
+  import TypesOfHistory._
+
+  object TypesOfStore extends Enumeration {
+    type TypeStore = Value
+    val Lmdb, InMemo = Value
+  }
+  import TypesOfStore._
```

Comment:
Using built in Scala 2 enumerations has some drawbacks but for what we are using it here, it's fine.

For the future here is more details. (we have Enumeratum in our dependencies)
https://pedrorijo.com/blog/scala-enums/
Scala 3 brings improvements.
https://dotty.epfl.ch/docs/reference/enums/enums.html

FYI @stanislavlyalin